### PR TITLE
Added logo and icons to sidebar - Mustafe

### PR DESCRIPTION
**Pull request by Muste8**
### Changes
Give a short summary of the changes made in this PR. 
-Added a new folder in assets/images/ that contains logo and icons
-Added logo and Icons to sidebar

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1206645498622417/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
-I didn't run any test for this task
-Couldn't find the logo to "My practice"

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

Screenshot 2024-03-28 at 9 54 54 AM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/layouts/_sidebar_nav.html.erb [hunk 1]
```diff
@@ -2,26 +2,30 @@
...
         <a href="#" class="nav-tab">
           <!--add my_students icon here-->
+          <img src="<%= asset_path("new_sidebar_icons/my_students_icon.png") %>" alt="my_students_icon" class="h-24 px-5"/>
+
           <i class="fa-brands fa-font-awesome"></i>
-          <span class="tab-title">My Students</span>
+          <span class="tab-title">My<br /> Students</span>
         </a>
       </li>
       <li>
```
Comment (camckin10) [inline app/views/layouts/_sidebar_nav.html.erb:18]: You're so close!

You are missing the 'My Learning' tab in the sidebar navigation. You already have the 'my_learning' icon. 
'My Learning' tab will be placed between the 'My Students' tab and 'My Community' tab. 

Apologies if I was not clear earlier; The only tabs that you and the other apprentices will be working on are the 'My Learning',  'My Students', and 'My Community' tabs.

Comment (camckin10) [reply app/views/layouts/_sidebar_nav.html.erb:18]: Once you add the 'my learning' tab to the navigation, then please update the screenshot included in your PR.

Comment (Muste8) [reply app/views/layouts/_sidebar_nav.html.erb:18]: Thanks for clarifying!

**Review by camckin10 — APPROVED** [review]
🔥  Looks amazing!

**Pull request merged**